fastlane/Fastfile

```diff
@@ -0,0 +1,169 @@
+# This is the minimum version number required.
+# Update this, if you use features of a newer version
+fastlane_version "2.146.0"
+
+skip_docs
+
+desc "Create a new FinniversKit version"
+lane :make_finniverkit_version do |options|
+  make_release("FinniversKit", "", options)
+end
+
+desc "Create a new FinnUI version"
+lane :make_finnui_version do |options|
+  make_release("FinnUI", "finnui-", options)
+end
+
+desc "Attempt to connect to github.com with ssh"
+lane :verify_ssh_to_github do
+  githuburl = "github.com"
+  UI.header "Verify ssh is setup and #{githuburl} can be accessed, will prompt for ssh key password if needed"
+  command = "ssh -T git@#{githuburl}"
+  sh(command, log: true, error_callback: ->(result) {
+    sshresult = result
+    if sshresult.include? "successfully authenticated"
+      UI.success "Could authenticate to #{githuburl}"
+    else
+      UI.shell_error! "Seems connection to #{githuburl} failed"
+    end
+    })
+  end
+
+  def make_release(project, tag_prefix, options)
+    ## Verify the branch.
+    branch = verify_branch("master")
+
+    ## Verify clean git status.
+    #ensure_git_status_clean
+
+    ## Create the new tag.
+    # If no bump in version is specified, only the release type version will be bumped. Ie. from 'rc1' to 'rc2'.
+    version_bump = lookup(options, :bump, %w(major minor patch))
+    if version_bump.nil?
+      version_bump = UI.select("What do you want to bump: ", ["major", "minor", "patch"])
+    end
+    last_version = version_get_podspec(path: "#{project}.podspec")
+    new_tag, new_version = new_tag_and_version(last_version, tag_prefix, version_bump)
+
+    ## Make sure the tag is unique.
+    ensure_tag_is_unique(new_tag)
+
+    ## Make the user verify that the tag is correct.
+    unless UI.confirm "Will create a release with verstion '#{new_version}' and tag '#{new_tag}'. Is that correct?"
+      UI.user_error! "Version was not accepted by user."
+    end
+
+    version_bump_podspec(path: "#{project}.podspec", version_number: new_version)
+
+    ## Update the version used when building
+    #todo: update_version_file("../#{project}/Version.h", new_version)
```

Comment:
Are these ones needed?

Reply:
No, not really, but my plan is to create these files so the build of the framework will actually specify the same version as the podspec, but can remove it until then